### PR TITLE
feat(cli): make minor improvements to speedtest

### PR DESCRIPTION
**Pull request by ammario**
- Remove mostly redundant "Transferred" column
- Rename "Bandwidth" to "Throughput"
- Replace "--reverse" (which has an ambiguous starting state) with "--direction=(up|down)"
- Tolerate AgentStartErrors which may be caused by failing startup script

### CODE REVIEW

File: cli/testdata/coder_speedtest_--help.golden [hunk 1]
```diff
@@ -4,11 +4,11 @@ Usage:
...
 
 Flags:
-  -d, --direct          Specifies whether to wait for a direct connection before testing speed.
-  -h, --help            help for speedtest
-  -r, --reverse         Specifies whether to run in reverse mode where the client receives and
-                        the server sends.
-  -t, --time duration   Specifies the duration to monitor traffic. (default 5s)
+  -d, --direct             Specifies whether to wait for a direct connection before testing speed.
+      --direction string   Specifies whether to run in reverse mode where the client receives
+                           and the server sends. (up|down) (default "down")
```
Comment (matifali) [inline cli/testdata/coder_speedtest_--help.golden:9]:
```suggestion
```

File: cli/testdata/coder_speedtest_--help.golden [hunk 2]
```diff
@@ -4,11 +4,11 @@ Usage:
   coder speedtest <workspace> [flags]
 
 Flags:
-  -d, --direct          Specifies whether to wait for a direct connection before testing speed.
-  -h, --help            help for speedtest
-  -r, --reverse         Specifies whether to run in reverse mode where the client receives and
-                        the server sends.
-  -t, --time duration   Specifies the duration to monitor traffic. (default 5s)
+  -d, --direct             Specifies whether to wait for a direct connection before testing speed.
+      --direction string   Specifies whether to run in reverse mode where the client receives
```
Comment (matifali) [inline cli/testdata/coder_speedtest_--help.golden:8]:
```suggestion
      --direction string   Specifies whether to test downlink or uplink throughput (up|down) (default "down")
```

Comment (ammario) [reply cli/testdata/coder_speedtest_--help.golden:8]: The CLI flag renderer should really be doing this.

**Comment by deansheather** [comment]
To update the golden files use `make update-golden-files`

**Review by deansheather — APPROVED** [review]

**Pull request merged**